test/cpp/jit/test_gpu.cpp

```diff
@@ -410,7 +411,7 @@ void testGPU_FusionClear() {
     tv1->computeAt(tv3, -1);
   }
 
-  prog.device_ = 0;
+  prog.getDevice() = 0;
```

Comment:
if you can set using this interface, please just use `prog.device()` if you really want it to be a method, otherwise use `getDevice()` and `setDevice(...)`

Reply:
I agree, no need for the "get" prefix. Even when we have setters, we can still use the plain name for getters (ex. device() + setDevice())